tests/unit/info-command.tcl

```diff
@@ -0,0 +1,105 @@
+start_server {tags {"info and its relative command"}} {
+    test "info command with only one argument" {
+        set info [r info]
+        assert { [string match "*redis_version*" $info] }
+        assert { [string match "*maxclients*" $info] }
+        assert { [string match "*used_cpu_user*" $info] }
+        assert { ![string match "*sentinel_tilt*" $info] }
+        assert { [string match "*used_memory*" $info] }
+        assert { [string match "*rdb_last_bgsave*" $info] }
+        assert { [string match "*master_repl_offset*" $info] }
+        assert { [string match "*cluster_enabled*" $info] }
+        assert { ![string match "*rejected_calls*" $info] }        
+    }
+
+    test "info all command" {
+        set info [r info all]
+        assert { [string match "*redis_version*" $info] }
+        assert { [string match "*maxclients*" $info] }
+        assert { [string match "*used_cpu_user*" $info] }
+        assert { ![string match "*sentinel_tilt*" $info] }
+        assert { [string match "*used_memory*" $info] }
+        assert { [string match "*rdb_last_bgsave*" $info] }
+        assert { [string match "*master_repl_offset*" $info] }
+        assert { [string match "*cluster_enabled*" $info] }
+        assert { [string match "*rejected_calls*" $info] }
```

Comment:
same here, this could be shorter, similar to the above, just expecting a different outcome for cmdstat